libs/etterlatte-oppgave-model/src/main/kotlin/OppgaveNy/OppgaveNy.kt

```diff
@@ -30,13 +31,21 @@ enum class Status {
 }
 
 enum class OppgaveType {
+    HENDELSE,
+    BEHANDLING,
+    EKSTERN
+}
+
+enum class Gjelder {
     FOERSTEGANGSBEHANDLING,
     REVURDERING,
+    MANUELT_OPPHOER,
     ATTESTERING,
     UNDERKJENT,
-    HENDELSE,
-    MANUELT_OPPHOER,
-    EKSTERN
+    KONTAKT_BRUKER,
+    VURDER_HENVENDELSE,
+    VURDER_KONSEKVENS,
+    GENERELL
 }
```

Comment:
Det skurrer litt med `Gjelder`-enum'en her, når vi dupliserer `behandlingType` inn (gjelder forsåvidt den gamle versjonen av `OppgaveType` også). Hvis (når?) vi skal vise f.eks. hendelsestype i oppgavelisten vil vi potensielt dytte inn disse og? 

Jeg ville likt en mer strukturert referanse f.eks., der vi holder på id + type, og så joiner mot aktuelle tabeller for å bygge opp DTO'en. men det er ikke noe vi trenger å løse nå med en gang

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1869/commits/8c612cc8e0962d29d03da6abd6255a11b974d42f

Reply:
Enig, endret til https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1869/commits/8c612cc8e0962d29d03da6abd6255a11b974d42f med kilde foreløpig. Men det er lettere å legge til joins senere enn å fjerne data som må migreres så vi ble enige om å gå for den siste commiten her med kilde nå også evt legge på joins senere.